runtime/parachains/src/paras.rs

```diff
@@ -765,7 +765,7 @@ pub mod pallet {
 
 		/// Includes a statement for a PVF pre-checking vote. Potentially, finalizes the vote and
 		/// enacts the results if that was the last vote before achieving the supermajority.
-		#[pallet::weight(0)]
+		#[pallet::weight(Weight::MAX)]
```

Comment:
Should we also create an issue to benchmark and replace with proper value, later ?

Reply:
Although it doesn't have a separate issue (yet) it is being tracked by https://github.com/paritytech/polkadot/issues/3211